website/server/controllers/api-v3/challenges.js

```diff
@@ -410,13 +541,17 @@ api.exportChallengeCsv = {
 };
 
 /**
- * @api {put} /api/v3/challenges/:challengeId Update a challenge
+ * @api {put} /api/v3/challenges/:challengeId Update a challenge. Only reflects
```

Comment:
I don't think this is true. According to the group model, you should be able to update `name`, `description`, `tasksOrder`, `leader`. Is that not the case?